client/src/components/App/Content/InvestigationForm/TabManagement/ExposuresAndFlights/ExposuresAndFlights.tsx

```diff
@@ -45,9 +43,15 @@ const ExposuresAndFlights: React.FC<Props> = ({ id }: Props): JSX.Element => {
     saveExposure(data , ids);
   }
 
+  const preventEnter = (e : React.KeyboardEvent) => {
```

Comment:
I'm guessing you are trying to prevent the enter key in the text fields to submit the form. 
Did you try using event.stopPropagation in the text field's events?

Reply:
If not it might solve this problem and this code can be removed.

Reply:
You can read more about it [here](https://developer.mozilla.org/en-US/docs/Web/API/Event/stopPropagation)

Reply:
I tried it initially, it didn't solve it  :/

Reply:
Did you figure out why it didn't work?

Reply:
first of all I got rid of preventEnter [here ](https://github.com/CoronaTeamMOD/CoronaInvestigation/pull/1025/commits/39979a97bb0c57ef435415bb682b9aac02957487)and went with just `e.preventDefault()` on the enter event
when I tried it initially and it was blocking all of the inputs , turns out I forgot to put e.preventDefault only when the 'Enter' key was pressed 😐.
stopPropagation did not in fact stop the propagetion and continued to submit the form , i guess this is the form's _default_ behavior

Reply:
Seems good 👍🏻